### ISSUE TITLE
Error returned: ModuleNotFoundError: No module named 'starlette.middleware.lifespan'

### ISSUE THREAD

**Issue by mikeckennedy** [kennethreitz/responder#300]
I cannot get responder to run the hello world app. I had it working a week or two ago but now there is an error that may have resurfaced. I've seen similar conversations discussed and closed.

Here we go:

Check out this console so you have the full history.

![screen shot 2019-02-19 at 12 29 58 pm](https://user-images.githubusercontent.com/2035561/53045510-a4dddc80-3442-11e9-8962-b486fab45faa.png)

Basically I:

* Installed responder with pipenv
* Created a file based on the quickstart code
* Activated the shell with pipenv
* Ran the project and got this error instead.

Surely I'm not missing something right? Just `pipenv install responder` and run it should work right?

**Comment by taoufik07** [comment]
Just stick with starlette==0.10.*

**Comment by mikeckennedy** [comment]
Thanks. That worked. Will there be a release soon were a straight install of responder works? Reason I ask is I'm doing some presentations around responder and want them to go OK. :)

**Comment by taoufik07** [comment]
@kenneth-reitz @timofurrer we need a new release ASAP please :)

**Comment by tomchristie** [comment]
I was under the assumption that responder is pinned to 0.10.x, which would mean this shouldn’t happen?

**Comment by taoufik07** [comment]
It's not released yet and starlette==0.10.6 "LisfespanHandler not found in routing.py "

**Comment by tomchristie** [comment]
I've reinstated the Starlette deprecations, which should resolve the issue.

@taoufik07 Once Responder has a PyPI release with Starlette pinned to a median version could you confirm to me. I'm totally happy to be responsible for making the tweaks needed between any deprecations, we just need Responder to pinned to `0.WHATEVER.*`.

* Issued a 0.11.2 which reinstates some deprecated API, putting it back in place until Responder is pinned.
* Issued a 0.10.7 release, which reinstates an usually internal bit of API.

**Comment by taoufik07** [comment]
There will be a new release tonight.
@tomchristie we leave it pinned to 0.10.5 or we pin it to >=0.10.7,<0.11 or >=0.11.2,<0.12 ?

**Comment by tomchristie** [comment]
0.10.* would make sense.
I’ll make the PR updating it to 0.11.* in due course.

**Comment by taoufik07** [comment]
Alright I will do a revert

**Comment by taoufik07** [comment]
New release **v1.3.0** :tada:

**Comment by mikeckennedy** [comment]
Thank you both! Works like a charm.

**Comment by Suriyakumardurai** [comment]
Hey guys.. 

instead of this 
from starlette.middleware.session import SessionMiddleware

use This 
from starlette_session import SessionMiddleware


before u have to install 
pip install starlette-session
